integration/config_test.go

```diff
@@ -56,12 +102,8 @@ func Test_ConfigAPI(t *testing.T) {
 				Timeout: 3 * time.Second,
 			}
 
-			requestDataPolicy := map[string]interface{}{}
-
-			requestDataJSON, _ := json.Marshal(requestDataPolicy)
-			requestDataBody := bytes.NewReader(requestDataJSON)
 			request, err := http.NewRequest(
-				"GET", fmt.Sprintf("http://localhost:9090/api/v1%s", tt.args.api), requestDataBody)
+				"GET", fmt.Sprintf("http://localhost:9090/api/v1/configs%s", tt.args.name), nil)
```

Comment:
Please change this to be 
```suggestion
				"GET", fmt.Sprintf("http://localhost:9090/api/v1/configs/%s", tt.args.name), nil)
```
and remove the slashes from `args.name`